### DISCUSSION TITLE
Where is the "Run" Command Located

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9657] [Q&A] [answered]
Hello - I am trying to locate where the "Run" command, for the "Run" menu item, is located. I have searched around but it is the only Menu Item on the main menu that I cannot find.

**Comment by vince-fugnitto** [comment, marked as answer]
@sboudreaux96 a quick search yields:

https://github.com/eclipse-theia/theia/blob/210fa4ac6875716f23ac70f94fe6f0ffc693bfe5/packages/debug/src/browser/debug-frontend-application-contribution.ts#L494

I'm not aware of a `run` command however, did you mean the `run` main-menu item like I've highlighted only?
